src/apps/create-patron-user-info/CreatePatron.tsx

```diff
@@ -28,21 +23,12 @@ const CreatePatron: FC = () => {
     }
 
     const {
-      attributes: { agencies, cpr: userCpr }
+      attributes: { cpr: userCpr }
     } = userInfo;
 
-    const userWasAlreadyCreated = agencies.some(
-      (agency) => agency.agencyId === agencyId
-    );
-
-    // If the user was already created, redirect to the dashboard.
```

Comment:
And we remove it from here, because we added this functionality into the entry file, right?

Reply:
yup